### PR TITLE
AS2 connector v5 User Guide

### PR DESCRIPTION
**Pull request by mcherem**
Version 5 of the AS2 connector for Mule 4. Please add Bharath Prakash as a reviewer.

### CODE REVIEW

File: modules/ROOT/pages/as2/as2-connectorMule4.adoc [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+|Self config - AS2 Partner Name| Sender’s AS2 system identifier. Used for AS2-From and AS2-To headers.
+|Self config - x509 alias| Sender’s Certificate and/or private key alias in the keystore. Required if using signing or encryption.
+|Self config - Email| Sender partner’s email.
+|Partner config - AS2 ID| Receiver’s AS2 system identifier. Used for AS2-From and AS2-To headers.
+|Partner config - x509 alias| Receiver’s Certificate and/or private key alias in the keystore. Required if using signing or encryption.
+|Partner config - Email| Receiver partner’s email.
+|Requester config - Subject| Free form text for MIME Subject header. Can include references to self / partner `simpleName` values along with the message `fileName`, for example, Sending file example.txt to partner via AS2.
+|Requester config - MDN MIC Signature Algorithm| The algorithm with which MIC is calculated for the MDN/receipt signature. `UNSIGNED` by default, that is, message is not signed.
+|Requester config - Request a receipt| Option to request a receipt from the partner. `SIGNED_OPTIONAL` by default.
+|Requester config - Request delivery URL| Asynchronous receipt delivery URL.
```
Comment (LukeSP) [inline modules/ROOT/pages/as2/as2-connectorMule4.adoc]:
```suggestion
|Requester config - Request delivery URL| Asynchronous receipt delivery URL which points to the MDN listener end point.
```

**Review by LukeSP — APPROVED** [review]

**Review by paigehargrave — APPROVED** [review]

File: modules/ROOT/pages/as2/as2-connector-upgrade-migrate.adoc [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+endif::[]
+:imagesdir: ../../assets/images/
+
+== Supported Migration Paths
+
+This a direct upgrade from AS2 connector 4.x to 5.0.0.
+
+== Changes in This Release
+
+* The `Async MDN` is now handled as a separate flow. The AS2 Message Send retry is not automatic. You need to add a new flow (with an endpoint) to handle async MDN.
```
Comment (bharathcp) [inline modules/ROOT/pages/as2/as2-connector-upgrade-migrate.adoc]:
```suggestion
* The `Async MDN` is now handled as a separate flow. You need to add a new flow (with an endpoint) to handle async MDN.
```

File: modules/ROOT/pages/as2/as2-connector-upgrade-migrate.adoc [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+
+Java 8
+
+=== Limitations
+
+Before you perform the migration, be aware of the following known limitations:
+
+* Signed copies of the document for nonrepudiation are not saved.
+
+* Authentication over HTTP is not supported.
```
Comment (bharathcp) [inline modules/ROOT/pages/as2/as2-connector-upgrade-migrate.adoc]: The present as2 v4 does not support Authentication over HTTP. So this limitation need not be a upgrade/migration concern.
```suggestion
```

File: modules/ROOT/pages/as2/as2-connector-upgrade-migrate.adoc [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+
+== Upgrade Prerequisites
+
+Before you perform the upgrade, you must complete the following:
+
+Do the users need to do anything before downloading the latest version?
```
Comment (bharathcp) [inline modules/ROOT/pages/as2/as2-connector-upgrade-migrate.adoc]: I cant think of anything.

File: modules/ROOT/pages/as2/as2-connector-upgrade-migrate.adoc [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+
+=== Upgrade Steps
+
+Follow these steps to perform the upgrade to version 5:
+
+. Download version 5 from Exchange.
+. Delete the old namespace and then add the new one.
++
+
+Before (AS2 v4 namespace):
```
Comment (bharathcp) [inline modules/ROOT/pages/as2/as2-connector-upgrade-migrate.adoc]:
```suggestion
==== Before (AS2 v4 namespace):
```

File: modules/ROOT/pages/as2/as2-connector-upgrade-migrate.adoc [hunk 6]
```diff
@@ -0,0 +1,257 @@
...
+----
+
+
+=== Post Upgrade Steps
+
+After you install the latest version of the connector, follow these steps to complete the upgrade:
+
+. Notify Partners of the new URL/port configuration that corresponds to the HTTP listener you are using.
+
+. Verify that the connector is working by ???
```
Comment (bharathcp) [inline modules/ROOT/pages/as2/as2-connector-upgrade-migrate.adoc]: Verify that the connector is working by making sure there are no exceptions on startup and testing against the AS2 partner.

**Review by bharathcp — APPROVED** [review]
approved with comments

**Pull request merged**